packages/block-editor/src/hooks/fit-text.scss

```diff
@@ -0,0 +1,3 @@
+.has-fit-text {
+	white-space: nowrap !important;
```

Comment:
Fit or Stretched Text can not wrap otherwise it does not make sense. But one can have multiple paragraphs on the same container.